src/FilterMonitor.js

```diff
@@ -10,10 +10,11 @@ export default class FilterMonitor extends Component {
     blacklist: PropTypes.array
   };
 
-  isFiltered(action) {
+  isFiltered(actionType) {
+    var type = actionType || '';
```

Comment:
I found where in our code, the type is missing. It was being deleted as we were using the action as the payload for our API call and deleting the type before sending it.